src/diffusers/loaders.py

```diff
@@ -962,13 +1014,14 @@ def lora_state_dict(
 
         # Convert kohya-ss Style LoRA attn procs to diffusers attn procs
         network_alpha = None
+        auxilary_states = ({}, {})
         if all((k.startswith("lora_te_") or k.startswith("lora_unet_")) for k in state_dict.keys()):
-            state_dict, network_alpha = cls._convert_kohya_lora_to_diffusers(state_dict)
+            state_dict, network_alpha, auxilary_states = cls._convert_kohya_lora_to_diffusers(state_dict)
```

Comment:
I have been thinking.

Do the CivitAI checkpoints always come from kohya trainers? Does it make sense to maybe rename this utility to `_convert_non_diffusers_lora_to_diffusers()`?

Reply:
As far as I know, there are three different formats:
- Kohya SS (which also has its variations for LyCORIS/LoCo/LoHO etc.)
- Diffusers
- Clone of Simo

We might be able to do it like that once we have the identify function discussed in https://github.com/huggingface/diffusers/issues/4061#issuecomment-1639516666